### PR TITLE
Feature: About page

### PR DESCRIPTION
**Pull request by guy196**

### CODE REVIEW

File: templates/aboutPage.html [hunk 1]
```diff
@@ -1,3 +1,78 @@
-<div>  
-  This is where you write all about yourself
-</div>
\ No newline at end of file
+<div>
+<style>
+body {
```
Comment (roeeyud) [inline templates/aboutPage.html:3]: Move all style to app.css

File: templates/index.html [hunk 2]
```diff
@@ -7,6 +7,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
         <link rel="stylesheet" href="/public/app.css">
         <link rel="shortcut icon" type="image/png" href="https://storage.cloud.google.com/printme3d_files/favicon.png">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (roeeyud) [inline templates/index.html:10]: What does this do?

Comment (guy196) [reply templates/index.html:10]: bootsrap scale

Comment (roeeyud) [reply templates/index.html:10]: 👍

File: templates/aboutPage.html [hunk 3]
```diff
@@ -1,3 +1,11 @@
-<div>  
-  This is where you write all about yourself
+<div>
+<div class="about-section">
```
Comment (roeeyud) [inline templates/aboutPage.html:2]: Missing about-section css code, did you forget or is this class not needed anymore?

File: templates/aboutPage.html [hunk 4]
```diff
@@ -1,3 +1,10 @@
-<div>  
-  This is where you write all about yourself
+<div class="container">
```
Comment (roeeyud) [inline templates/aboutPage.html:1]: nice

File: public/app.css [hunk 5]
```diff
@@ -18,17 +18,17 @@
...
+}
 .card-img-contain {
   text-align: center;
   height: 200px;
 }
-
 .product-row {
   justify-content: center;
 }
-h1 {
```
Comment (roeeyud) [inline public/app.css:29]: Deleting code is always worth extra point! and a gif

**Pull request merged**